### PR TITLE
Feature/sig 890 dashboard

### PR DESCRIPTION
**Pull request by jasperswart**

### CODE REVIEW

File: src/signals/incident-management/containers/DashboardContainer/components/CategoryChart/style.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.category-graph {
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/components/CategoryChart/style.scss]: This className is not defined anywhere, should it be category-chart?

File: src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+}
+
+const colors = {
+  'Gemeld': 'red',
+  'Afgehandeld': 'lightgreen',
+  'Heropend': 'orange',
+  'In afwachting van behandeling': 'purple',
+  'In behandeling': 'blue',
+  'On hold' : 'grey',
+  'Geannuleerd' : 'darkgrey'
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js]: Better to use the defined constants for the statuses (statusList.js)

File: src/signals/incident-management/containers/DashboardContainer/components/StatusChart/style.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.status-graph {
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/components/StatusChart/style.scss]: Should be status-chart?

File: src/signals/incident-management/containers/DashboardContainer/reducer.test.js [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+          payload
+        }).toJS()
+      ).toEqual({
+        loading: false,
+        dashboard: payload
+      });
+    });
+  });
+
+  describe('REQUEST_CATEGORY_UPDATE_SUCCESS', () => {
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/reducer.test.js]: These 3 actions are not the ones imported

File: src/signals/incident-management/containers/DashboardContainer/reducer.test.js [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+        }).toJS()
+      ).toEqual({
+        loading: false,
+        error: true
+      });
+    });
+  });
+
+  describe('UPDATE_DASHBOARD', () => {
+    it('sets loading and error', () => {
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/reducer.test.js]: This isn't tested here.

File: src/signals/incident-management/containers/DashboardContainer/saga.js [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+export function* fetchDashboard() {
+  try {
+    const dashboard = yield authCall(requestURL);
+
+    dashboard.hour = dashboard.hour.map((item) => ({
+      ...item,
+      timestamp: new Date(item.interval_start).getTime()
+    }));
+
+    // TEMP: remove these items
```
Comment (RobertLodder) [inline src/signals/incident-management/containers/DashboardContainer/saga.js]: Should this be removed later or now?

File: src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js [hunk 7]
```diff
@@ -0,0 +1,58 @@
...
+import { PieChart, Pie, Cell, Tooltip, Legend } from 'recharts';
+
+import './style.scss';
+
+const RADIAN = Math.PI / 180;
+
+function renderStatusLabel({ cx, cy, midAngle, innerRadius, outerRadius, percent, index }) {
+ 	const radius = innerRadius + (outerRadius - innerRadius) * 0.5;
+  const x  = cx + radius * Math.cos(-midAngle * RADIAN);
+  const y = cy  + radius * Math.sin(-midAngle * RADIAN);
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js]: gedaaan

File: src/signals/incident-management/containers/DashboardContainer/components/CategoryChart/style.scss (hunk 1 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/components/CategoryChart/style.scss]: gedaan

File: src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js (hunk 2 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/components/StatusChart/index.js]: veel beter gedaan

File: src/signals/incident-management/containers/DashboardContainer/components/StatusChart/style.scss (hunk 3 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/components/StatusChart/style.scss]: gedaan

File: src/signals/incident-management/containers/DashboardContainer/reducer.test.js (hunk 4 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/reducer.test.js]: gedaan

File: src/signals/incident-management/containers/DashboardContainer/reducer.test.js (hunk 5 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/reducer.test.js]: gedaan

File: src/signals/incident-management/containers/DashboardContainer/saga.js (hunk 6 above)
Comment (jasperswart) [reply src/signals/incident-management/containers/DashboardContainer/saga.js]: we do not get the timestamp from the server. only date string. what do you suggest?

**Pull request merged**